src/locales/en.ts

```diff
@@ -93,6 +93,7 @@ const en = {
     form: {
       bodyPlaceholder: () => "Post something awesome...",
     },
+    imageUploadError: () => "Unable to upload image(s)"
```

Comment:
Should we move this into `form`?

Reply:
Still within posts I assume? That's where I had it initially and it makes sense. I'll move it over now.

Reply:
Oh, good question! I think this might actually make more sense in `forms` at the top most level, since this can apply for other models as well